marshalable-derive/Cargo.toml

```diff
@@ -10,3 +10,5 @@ proc-macro = true
 proc-macro2 = { workspace = true }
 quote = { workspace = true }
 syn = { workspace = true }
+log = "0.4.22"
```

Comment:
why are are having those, you do not depend on log at all! also depending on log in proc macro is generally undesirable, rust can already propagate error as compiler error

Reply:
I forgot to remove this!! It was used for dumping syntax tree at compile time, to support during development. Removed again.